packages/adminrouter/extra/src/includes/server/windows-agent.conf

```diff
@@ -0,0 +1,3 @@
+location / {
+    return 404;
+}
```

Comment:
This is the same as the standard `server/agent.conf`. Can we use the same file for this case?

Reply:
Done